src/patterns/dosdp-patterns/isolated.yaml

```diff
@@ -1,5 +1,10 @@
 pattern_name: isolated
 
+description: >-
+    A disease that has an isolated presentation, that is not part of a larger syndrome.
```

Comment:
This is good, but in general I'd recommend a shift in emphasis, as we are defining the pattern not the disease, and give some general context as to why the pattern exists.

something like:

------

Some diseases exist in both isolated and syndromic forms. For example, aniridia. Use this pattern to define the isolated form of a disease when a term exists for the isolated/syndromic-neutral version.

In general, this pattern should be used in parallel with syndromic. E.g. if you make a term isolated D, you should also have syndromic D. (link to syndromic pattern)

Note that the isolated and syndromic forms will be inferred to be disjoint due to the GCI pattern (link to docs on GCIs)

-------

@matentzn no direct action but we need to be thinking of things like this in the pattern framework. We want consistent population of patterns, e.g. isolated-D accompanied by syndromic-D. We also want a nice way of interlinking in the docs

Reply:
@cmungall thanks for the feedback, I updated accordingly.

What is the GCI pattern?
It is mentioned here too, without definition:
https://mondo.readthedocs.io/en/latest/editors-guide/e-design-patterns/#isolated-vs-syndromic-diseases

Reply:
Not sure what the docs on GCIs are